spec/datadog/profiling/validate_benchmarks_spec.rb

```diff
@@ -20,4 +20,8 @@
   describe 'profiler_sample_serialize' do
     it('runs without raising errors') { expect_in_fork { load './benchmarks/profiler_sample_serialize.rb' } }
   end
+
+  describe 'tracing_trace' do
+    it('runs without raising errors') { expect_in_fork { load './benchmarks/tracing_trace.rb' } }
+  end
```

Comment:
Minor: Placing this test here means it's also covered by the `skip_if_profiling_not_supported` e.g. it probably will be skipped on most folks' machines because you need to manually build libdatadog for profiler to run on macOS.

I would suggest either introducing a `context 'profiling benchmarks'` to contain the profiling stuff, and thus allowing this benchmark to be outside of it (I guess in the process also moving this file to `spec/datadog/`?), or alternatively introduce a separate `tracing/validate_benchmarks_spec.rb` that does not need the profiling checks at all.

P.s.: I guess we can remove the `, if: (RUBY_VERSION >= '2.4.0')` on line 3? :)

Reply:
I didn't notice the file location, sorry about that!
I created `tracing/validate_benchmarks_spec.rb` as suggested.